### ISSUE TITLE
Facebook 'view all [X] comments' does not work properly

### ISSUE THREAD

**Issue by int3** [philc/vimium#134]
Using vimium to click on the abovementioned link causes the browser to open up http://www.facebook.com/ajax/ufi/modify.php instead of displaying the rest of the comments.

Actually, the 'view all [X] comments' appears to be a submit button disguised (styled) as a link. It's within a form that's set to POST to modify.php. I suppose there's something like event.preventDefault going on in the background, but somehow it isn't being triggered when we create our own mouse events. Unfortunately, there is so much obfuscated javascript powering fb that it's hard to isolate the problem. On the other hand, my own small testcases don't trigger the issue.

I suspect that it may be a Chrome bug -- I tinkered with vimperator to make it use more or less the same link-clicking code as we do, and it still gives the correct behavior. Still, it would be great if we could narrow down the bug with more test cases. (I'm also kind of interested in hacking on webkit / chrome -- I might try this myself if it isn't too difficult.)

tl;dr: hope to find more test cases (:
